build.sh

```diff
@@ -288,10 +288,6 @@ if [ -z "$configuration" ]; then
 fi
 msbuild_args[${#msbuild_args[*]}]="-p:Configuration=$configuration"
 
-# Set verbosity
-echo "Setting msbuild verbosity to $verbosity"
-msbuild_args[${#msbuild_args[*]}]="-verbosity:$verbosity"
-
```

Comment:
If I didn't remove this, we end up with extra options on the `msbuild` command line because tools.sh handles `$verbosity` itself. Doesn't matter much but also doesn't help at all.